### PR TITLE
What's new: issue number edits, recognize Windows 10 RS5/Server 2019

### PR DESCRIPTION
**Pull request by josephsl**
### Link to issue number:
None

### Summary of the issue:
Edits to what's new document for NVDA 2018.3.

### Description of how this pull request fixes the issue:
Changed issue numbers where they were previously pointing to pull requests. In case of braille tables, the two entries have been combined into one, as well as capitalized table names. Also, recognize the fact that Windows 10 RS5 has a server variant: Windows Server 2019.

### Testing performed:
Compiled docs from source with no issues.

### Known issues with pull request:
None

### Change log entry:
None

Thanks.

### CODE REVIEW

File: user_docs/en/changes.t2t [hunk 1]
```diff
@@ -7,12 +7,11 @@ What's New in NVDA
...
 == New Features ==
 - NVDA will report grammar errors when appropriately exposed by web pages in Mozilla Firefox. (#8280)
-- Added support for BrailleNote QT and Apex BT's scroll wheel when BrailleNote is used as a braille display with NVDA. (#6316)
+- Added support for BrailleNote QT and Apex BT's scroll wheel when BrailleNote is used as a braille display with NVDA. (#5992, #5993)
 - Added scripts for reporting elapsed and total time of current track in Foobar2000. (#6596)
 - The Mac command key symbol (⌘) is now announced when reading text with any synthesizer. (#8366)
-- custom roles via the aria-roledescription attribute are now supported in all web browsers. (#8449)
-- New braille table: Swedish 8 dot computer braille. (#8227)
-- added czech eight dots, central kurdish, esperanto and hungarian braille tables. (#8446)
+- custom roles via the aria-roledescription attribute are now supported in all web browsers. (#8448)
```
Comment (leonardder) [inline user_docs/en/changes.t2t]: Please change this line to start with an uppercase c

File: user_docs/en/changes.t2t [hunk 2]
```diff
@@ -7,12 +7,11 @@ What's New in NVDA
...
+- New braille table: Czech 8 dot, Central Kurdish, Esperanto, Hungarian, Swedish 8 dot computer braille. (#8226, #8437)
```
Comment (leonardder) [inline user_docs/en/changes.t2t]: table>tables

File: user_docs/en/changes.t2t [hunk 3]
```diff
@@ -41,8 +40,8 @@ What's New in NVDA
 - Pressing sp2 on an ALVA BC6 display to emulate the alt key now works as advertised. (#8360)
 - NVDA no longer announces redundant keyboard layout changes. (#7383)
 - Mouse tracking is now much more accurate in Notepad and other plain text edit controls when in a document with more than 65535 characters. (#8397)
-- NVDA will recognize more dialogs in Windows 10 and other modern applications. (#7473)
-- On windows 10 rs5 and above, NVDA no longer fails to track the system focus when an application freezes or floods the system with events. (#8539)
+- NVDA will recognize more dialogs in Windows 10 and other modern applications. (#8405)
+- On Windows 10 rs5 and Server 2019 and above, NVDA no longer fails to track the system focus when an application freezes or floods the system with events. (#7345, #8535)
```
Comment (leonardder) [inline user_docs/en/changes.t2t]: I wonder whether we should refer to rs5 as the Windows 10 Oktober 2018 update, which will most likely be the name for that update.

**Review by michaelDCurran — DISMISSED** [review]

**Review by LeonarddeR — CHANGES_REQUESTED** [review]
Just to make sure, could you check that every entry starts with a capital letter? We should actually unit test this somehow.

Furthermore, it might be good to add an entry for 2018.2.1, not that it's super important, but it will make it easier to find out what this release was for in the future.

**Comment by josephsl** [comment]
Hi, regarding RS5 name, perhaps that might work, but we don’t want to go through yet another Spring Creators Update debacle like what we saw earlier this year. I’m happy for that to be called October 2018 Update for now. Thanks.

**Review by michaelDCurran — DISMISSED** [review]

**Comment by michaelDCurran** [comment]
@leonardder are you happy with this now?

File: user_docs/en/changes.t2t [hunk 4]
```diff
@@ -8,29 +8,28 @@ What's New in NVDA
...
 - Content marked as being either inserted or deleted in web pages is now reported in Google Chrome. (#8558)
-- Added support for BrailleNote QT and Apex BT's scroll wheel when BrailleNote is used as a braille display with NVDA. (#6316)
+- Added support for BrailleNote QT and Apex BT's scroll wheel when BrailleNote is used as a braille display with NVDA. (#5992, #5993)
 - Added scripts for reporting elapsed and total time of current track in Foobar2000. (#6596)
 - The Mac command key symbol (⌘) is now announced when reading text with any synthesizer. (#8366)
-- custom roles via the aria-roledescription attribute are now supported in all web browsers. (#8449)
-- New braille table: Swedish 8 dot computer braille. (#8227)
-- added czech eight dots, central kurdish, esperanto and hungarian braille tables. (#8446)
+- Custom roles via the aria-roledescription attribute are now supported in all web browsers. (#8448)
+- New braille table: Czech 8 dot, Central Kurdish, Esperanto, Hungarian, Swedish 8 dot computer braille. (#8226, #8437)
```
Comment (leonardder) [inline user_docs/en/changes.t2t]: table>tables

**Review by LeonarddeR — DISMISSED** [review]
Go ahead after fixing just one mistake

**Comment by josephsl** [comment]
Hi, done. Thanks.

**Review by michaelDCurran — APPROVED** [review]

**Pull request merged**